src/components/side-menu/SideMenu.tsx

```diff
@@ -0,0 +1,27 @@
+import {Divider, Menu} from 'antd'
+import React from 'react'
+
+export interface MenuEntry {
+  name: string
+}
+
+export interface SideMenuProps {
+  entries: MenuEntry[]
+}
+
+const dividerStyle: React.CSSProperties = {margin: 0, width: '80%', minWidth: '80%', float: 'right'}
+
+export const SideMenu: React.FC<SideMenuProps> = ({entries}) => {
+  const entriesMapper = (entry: MenuEntry) => (
+    <React.Fragment key={entry.name}>
+      <Menu.Item className="menu-entry" key={entry.name}>{entry.name}</Menu.Item>
```

Comment:
I think that is not allowed to have 2 items with the same key. You already used `entry.name` as key in the previous line